cms-plugin/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/content/bulk/bulk-result.jsp

```diff
@@ -2,98 +2,88 @@
-        <li><s:text name="breadcrumb.app" /></li>
-        <li><s:text name="breadcrumb.jacms" /></li>
-        <li>
-            <a href="<s:url action="list" namespace="/do/jacms/Content"/>">
-                <s:text name="breadcrumb.jacms.content.list" />
-            </a>
-        </li>
-        <li class="page-title-container"><s:property value="%{#labelTitle}" /></li>
-    </ol>
+<!-- Admin console Breadcrumbs -->
+<ol class="breadcrumb page-tabs-header breadcrumb-position">
+    <li><s:text name="breadcrumb.app" /></li>
+    <li><s:text name="breadcrumb.jacms" /></li>
+    <li>
+        <a href="<s:url action="list" namespace="/do/jacms/Content"/>">
+            <s:text name="breadcrumb.jacms.content.list" />
+        </a>
+    </li>
+    <li class="page-title-container"><s:property value="%{#labelTitle}" /></li>
+</ol>
 
-    <!-- Page Title -->
-    <s:set var="dataContent" value="%{'help block'}" />
-    <s:set var="dataOriginalTitle" value="%{'Section Help'}"/>
-    <h1 class="page-title-container">
-        <s:property value="%{#labelTitle}" />&#32;&ndash;&#32;<s:text name="title.bulk.result" />
-    </h1>
+<!-- Page Title -->
+<s:set var="dataContent" value="%{'help block'}" />
+<s:set var="dataOriginalTitle" value="%{'Section Help'}"/>
+<h1 class="page-title-container">
+    <s:property value="%{#labelTitle}" />&#32;&ndash;&#32;<s:text name="title.bulk.result" />
+</h1>
 
-    <!-- Default separator -->
-    <div class="form-group-separator"></div>
+<!-- Default separator -->
+<div class="form-group-separator"></div>
 
-    <div id="main" role="main">
+<div id="main" role="main">
 
-        <div class="alert alert-success mt-20">
-            <span class="pficon pficon-ok"></span>
-
-            <s:set var="report" value="#command.report" />
-            <p>
-                <strong><s:text name="label.bulk.status" /></strong>: <s:text name="name.bulk.status.%{#command.status}" />
-            </p>
-            <s:if test="%{#report.endingTime != null}" >
-                <p>
-                    <strong><s:text name="label.bulk.report.endingTime" /></strong>: <s:date name="#report.endingTime" format="dd/MM/yyyy HH:mm:ss"/>
-                </p>
-            </s:if>
-            <p>
-                <strong><s:text name="label.bulk.report.total" /></strong>:
-                <s:text name="label.bulk.report.total.count" >
-                    <s:param name="applyTotal" value="%{#report.applyTotal}" />
-                    <s:param name="total" value="%{#report.total}" />
-                </s:text>
-            </p>
-            <p>
-                <strong><s:text name="label.bulk.report.successCount" /></strong>: <s:property value="%{#report.applySuccesses}" />
-            </p>
+    <div class="alert alert-success mt-20">
+        <span class="pficon pficon-ok"></span>
+        <%--
+        <p>
+            <strong><s:text name="label.bulk.status" /></strong>: <s:text name="name.bulk.status.%{#command.status}" />
```

Comment:
Can we remove these commented lines?

Reply:
there was a comment in the original code in entando-ps, i will remove it

Reply:
done